setup.py

```diff
@@ -31,7 +31,6 @@
     "bitarray~=2.3.7": ["install"],
     "bump2version~=1.0.1": ["dev"],
     "click~=8.0.4": ["install"],
-    "cupy-cuda113~=10.4.0": ["install"],
```

Comment:
I'm not sure what removing 
`cupy-cuda113~=10.4.0": ["install"],`
will do to the rest of the components in PrimeQA, but the CI tests are passing.